utils/update.ts

```diff
@@ -53,28 +51,28 @@ export async function update() {
       }
 
       if (currentVersion !== latestVersion) {
-        console.info(
-          `Upgrading ${pkg} ${currentVersion} -> ${latestVersion}.`,
-        );
+        console.info(`Upgrading ${pkg} ${currentVersion} -> ${latestVersion}.`);
 
         upgradeFound = true;
         importMap.imports[pkg] = url.at(latestVersion).url;
       }
 
       return importMap;
     },
-    Promise.resolve(Object.assign({ imports: {} }, importMap)),
+    Promise.resolve(Object.assign({ imports: {} }, importMap))
   );
 
+  if (!newImportMap.imports["deco/"] && newImportMap.imports["$live/"]) {
```

Comment:
Check this out => https://github.com/deco-cx/deco/blob/main/scripts/upgrade.ts#L408

Reply:
Its different scripts and in both "deco/' alias is required. Anyway, in update script it will be add just if its doesnt exists and in upgrade script its be add using a JS obejct, wich means any duplicated key will be override.